pkg/apis/v1beta1/cluster.go

```diff
@@ -175,20 +185,27 @@ func DefaultAPISpec() *APISpec {
 	// Collect all nodes addresses for sans
 	addresses, _ := util.AllAddresses()
 	publicAddress, _ := util.FirstPublicAddress()
+	addresses = append(addresses, publicAddress)
```

Comment:
why do we need to add first public address to `AllAddresses`?

Reply:
I believe it was there before Jussi only moved it up and filtered it with util.Unique

Reply:
uh, we don't. `AllAddresses()` should give us all addresses already. I'll fix this